src/Main.java

```diff
@@ -18,14 +18,18 @@ public static void main(String[] args) {
 //Задача 2
         int clientDeviceYear=2015;
         int clientOsOne=1;
-        if (clientDeviceYear<=2015 && clientOsOne>=1) {
+        if (clientDeviceYear<2015 && clientOsOne>=1) {
             System.out.println("Установите облегченную версию приложения для iOS по ссылке");
-        } else {
+        } else if (clientDeviceYear<2015 && clientOsOne>=0) {
```

Comment:
```suggestion
        } else if (clientDeviceYear<2015 && clientOsOne==0) {
```
clientOsOne>=0 ? должно быть равно